backend/question-service/package.json

```diff
@@ -4,20 +4,23 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "nodemon ./src/server.ts"
```

Comment:
Better have a `start` script for deployment as well.

Reply:
yeah , will push will the Dockerisation